src/api/app/assets/javascripts/webui/comment.js

```diff
@@ -29,19 +29,55 @@ $(document).ready(function(){
     resizeTextarea(this);
   });
 
+  $('.timeline').on('keyup', '.comment-field', function(e) {
```

Comment:
To reduce the amount of code, we could combine the various `on` calls like this one here would become:

```suggestion
  $('.timeline', '.comment-new').on('keyup', '.comment-field', function(e) {
```

Reply:
I was about to suggest having two different files, one for the legacy and another one for the beta.
This would mean a lot of duplication but simplicity to read the code now and to remove the legacy code later.

Reply:
We don't have any concrete plans yet to change how the comments on other page will behave. I guess we will do it eventually, so the separation would be beneficial :+1:.

Reply:
Ah, I just went and merged all the selectors...

Reply:
That's ok